opm/grid/cpgrid/Intersection.cpp

```diff
@@ -121,12 +125,17 @@ int Intersection::indexInInside() const
     const bool normal_is_in = !f.orientation();
     enum face_tag tag = pgrid_->face_tag_[f];
     switch (tag) {
-    case LEFT:
+    case I_FACE:
         return normal_is_in ? 0 : 1; // min(I) : max(I)
-    case BACK:
+    case J_FACE:
         return normal_is_in ? 2 : 3; // min(J) : max(J)
-    case TOP:
+    case K_FACE:
         return normal_is_in ? 4 : 5; // min(K) : max(K)
+    case NNC_FACE:
+        // For nnc faces we return the otherwise unused value -1.
+        // The Dune grid interface is essentially meaningless
+        // for non-neighbouring "intersections".
+        return -1;
```

Comment:
I doubt, that this works with the ewoms discretization.

Reply:
Indeed, this required a matching change in ecltransmissibilities.hh, as can be seen in the companion PR. It only required that one change though.